src/sections/Projects/Sistent/components/link/guidance.js

```diff
@@ -0,0 +1,171 @@
+          </p>
+          <a id="Function">
+            <h2>Function</h2>
+          </a>
+          <p>
+            Links too have a lot many functions as:
+          </p>
+          <h3>Navigation Links</h3>
+          <p>
+          These links help users navigate through a website, directing them to important sections or related content. They should be easily accessible and clearly labeled to enhance usability.
+          </p>
+          <Row $Hcenter className="image-container">
+            <SistentThemeProvider initialMode={isDark ? "dark" : "light"}>
+              <Link
+                href="https://layer5.io/projects/sistent/components/link/guidance" // Specify the path you want to navigate to
+                style={{
+                  marginLeft: "10px",
+                  textDecoration: "none",
+                  color: "green", // Change color based on the theme
+                  padding: "6px",
+                }}
+              >
+                Navigate to Layer 5
+              </Link>
+            </SistentThemeProvider>
+          </Row>
+          <h3>Call to Action (CTA) Links</h3>
+          <p>
+          CTA links encourage users to take specific actions, such as signing up for a newsletter or downloading a resource. They should stand out visually to attract user attention and drive engagement.
+          </p>
+          <Row $Hcenter className="image-container">
+            <SistentThemeProvider initialMode={isDark ? "dark" : "light"}>
+              <Link
+                href="https://layer5.io/projects/sistent/components/link/guidance"
+                style={{
+                  display: "inline-block",
+                  marginLeft: "10px",
+                  textDecoration: "none",
+                  color: "black",
+                  backgroundColor: "yellow",
+                  padding: "10px 20px",
+                  borderRadius: "4px",
+                  fontWeight: "bold",
+                  border: "2px solid black",
+                }}
+              >
+                Get Started
+              </Link>
+            </SistentThemeProvider>
+          </Row>
+          <h3>External Links</h3>
+          <p>
+          These links direct users to external websites. It’s important to provide clear indicators (like icons or different styles) that these links lead to external content, which helps users manage their navigation expectations.
+          </p>
+          <Row $Hcenter className="image-container">
+            <SistentThemeProvider initialMode={isDark ? "dark" : "light"}>
+              <Link
+                href="https://layer5.io/projects/sistent/components/link/guidance"
+                target="_blank"
+                rel="noopener noreferrer"
+                style={{
+                  marginLeft: "10px",
+                  textDecoration: "underline",
```

Comment:
external link should have underline by default , it should be on hover.
and will you use this icon also with external link. We use for our external links
![image](https://github.com/user-attachments/assets/d5833939-6683-4261-a866-8202b10e2b22)

Reply:
you will find in sistent

Reply:
Okay sir I will do this change